generators/ruby/faraday/templates/gemspec.mustache

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
 
   {{#isFaraday}}
   s.add_runtime_dependency 'faraday', '>= 1.0.1', '< 3.0'
+  s.add_runtime_dependency 'faraday-follow_redirects', '~> 0.3.0'
```

Comment:
@Kallin thanks for your contribution, it LGTM. I'm only suggesting below change for tracking purpose:
```suggestion
  s.add_runtime_dependency 'faraday-follow_redirects', '~> 0.3.0'   {{! added dependency }}
```